Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ruff = "*"
+pycodestyle = "*"
```

Comment:
We shouldn't be using `pycodestyle` anymore.

Reply:
Right, removing.